code/turf/turf_flock.dm

```diff
@@ -359,6 +359,7 @@ turf/simulated/floor/feather/proc/bfs(turf/start)//breadth first search, made by
 		src.desc = "It's broken. You could probably use a crowbar to break the pieces apart."
 		src.broken = TRUE
 		src.UpdateIcon()
+		src.material.setProperty("reflective", 25)
```

Comment:
Reading material code this probably changes the material's reflective value. everywhere.

Reply:
I tested reflecting projectiles off other walls after breaking a wall, it doesn't seem to mess with other walls